src/ops.rs

```diff
@@ -125,7 +125,89 @@ pub fn jcc(vm: &mut VM, pipeline: &Pipeline, hv: &mut dyn Hypervisor) -> Result<
     Ok(())
 }
 
-pub fn mul_8bit(vm: &mut VM, pipeline: &Pipeline, _hv: &mut dyn Hypervisor) -> Result<(), VMError> {
+pub fn div_8bit(vm: &mut VM, pipeline: &Pipeline, hv: &mut Hypervisor) -> Result<(), VMError> {
+    let first_arg = vm.reg16(Reg16::AX) as u16;
+    let second_arg = vm.get_arg(pipeline.args[0].location)?.u16_zx()?;
+    if second_arg == 0 || ((first_arg / second_arg) > 0xFF) {
+        return Err(VMError::DivideByZero) // divide by 0 not allowed and result being too big for destination not allowed
```

Comment:
Should this be broken out into single errors (ie, DivideByZero and maybe DestinationTooSmall or some other error)? I know x86 treats both internally as divide by zero, but this might make things harder to debug